framework/db/ActiveQuery.php

```diff
@@ -81,6 +81,11 @@ class ActiveQuery extends Query implements ActiveQueryInterface
     const EVENT_INIT = 'init';
 
     /**
+     * string placeholder to be replaced by the table alias of a query.
+     */
+    const ALIAS_PLACEHOLDER = '@alias';
```

Comment:
Thanks for your feedback.
However, I have a hard time to understand what you are trying to point out. 
Could you be more explicit?